### PR TITLE
Vim syntax layer.

### PR DESCRIPTION
**Pull request by ralesi**
Vim and pentadactyl syntax highlighting

### CODE REVIEW

File: layers/!lang/vim/packages.el [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    :init
+    (progn
+      (add-hook 'vimrc-mode-hook '(lambda () 
+                                    (highlight-numbers-mode -1)
+                                    (rainbow-delimiters-mode-disable))))))
+
+(defun vim/init-dactyl-mode ()
+  (use-package dactyl-mode
+    :mode "pentadactylrc\\'"
+    :mode "\\.penta\\'"
```
Comment (TheBB) [inline layers/!lang/vim/packages.el]: Might as well add vimperator files in here... shouldn't make a huge difference, no?

**Comment by ralesi** [comment]
I will check if there is a Melpa package for it.

**Comment by ralesi** [comment]
@TheBB .  Can't find any vimperator package. I can assign Vimperator files to the dactyl package, but I think thats the best we get.   Should I add them?

**Comment by TheBB** [comment]
That's what I had in mind, yeah. I don't think there's major differences.

**Comment by ralesi** [comment]
@TheBB updated with vimperator filetypes as well.

**Comment by ralesi** [comment]
Let me know if you need this rebased.

**Comment by syl20bnr** [comment]
Thank you ! :+1: 
I renamed the layer `vimscript` to be more explicit.
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**